src/App.jsx

```diff
@@ -1,19 +1,96 @@
 import logo from "/logo.png";
 import "./App.css";
+import React, { useEffect } from "react";
+import { Container } from "react-bootstrap";
+import { Row } from "react-bootstrap";
+import { Col } from "react-bootstrap";
+import { getPokemonImage, getRandomPokemon } from "./utils";
+import { useState } from "react";
+import { pokedex } from "./pokedex";
+import { UserGuess } from "./component/userguess";
+import { ScramblePokeName } from "./component/scramblename";
+import { CheckWin } from "./component/checkwin";
+import Button from "react-bootstrap/Button";
 
 function App() {
+  //generating a random pokemon name from the json file.
+
+  const [currPokemon, setCurrentPokemon] = useState(getRandomPokemon());
+  const [guessedPokemon, setGuessedPokemon] = useState([]);
+  const [currGuess, setCurrGuess] = useState("-");
+  const [score, setScore] = useState(0);
+  const [hpLeft, setHpLeft] = useState(6);
+
+  //7. When the round ends, give the user an option to play again or restart the game.
```

Comment:
This is the first thing I read, but labeled with 7. - confusing! Also, then name `handleRestart` captures this information already